eslint.config.mjs

```diff
@@ -466,19 +445,76 @@ export default tseslint.config([
           shorthandLast: true,
         },
       ],
+      '@stylistic/jsx-tag-spacing': [
+        'error',
+        {
+          afterOpening: 'never',
+          beforeClosing: 'allow',
+          beforeSelfClosing: 'always',
+          closingSlash: 'never',
+        },
+      ],
       '@stylistic/lines-between-class-members': [
         'error',
         'always',
         { exceptAfterSingleLine: true },
       ],
       '@stylistic/no-tabs': 'error',
+      '@typescript-eslint/no-unused-vars': [
+        'error',
+        {
+          args: 'after-used',
+          argsIgnorePattern: '^_',
+          varsIgnorePattern: '^_',
+        },
+      ],
       // Blocks double-quote strings (unless a single quote is present in the
       // string) and backticks (unless there is a tag or substitution in place).
       '@stylistic/quotes': ['error', 'single', { avoidEscape: true }],
 
       // The _.omit function is still useful in some contexts.
       'you-dont-need-lodash-underscore/omit': 'off',
     },
+
+    settings: {
```

Comment:
This is a nit: I don't know that I agree with this placement. Conventionally all the settings come first, and then all the rules. The rule is a good starting point for this file, but I don't think it should be blanket applied and I think we should retain control over the ordering of the top-level object.

Reply:
A couple options: 

- A comment will prevent reordering
- Wrapping only part of the object in `eslint-disable/enable sort-objects` will prevent reordering 
- Waiting for https://github.com/azat-io/eslint-plugin-perfectionist/pull/559 which should have syntax like `// @sort-keys`